ci/release-info-v3.py

```diff
@@ -0,0 +1,100 @@
+import json
+import os
+import re
+import sys
+from datetime import date
+from subprocess import Popen, PIPE
+
+def check_version(ver):
+    # Checks package version for matching with the current tag reference
+    # print(ver[0])
+    if ref is not None and ref != 'refs/tags/v' + str(ver[0]):
+        return 0
+    else:
+        return 1
+
+def print_error(crate, ver, v):
+    # print(crate, ver[0], v)
+    if not v:
+        print(
+            '::error file={path}::version {0} does not match release tag {1}'
+            .format(str(ver[0]), ref, path = str(crate) + '/Cargo.toml')
```

Comment:
This will not produce a correct path, because some crates live under `./testing`.

Reply:
Sure, will rework that - will retrieve paths from manifest json data, 'cause in v3 script the paths are can be taken only from metadata, that's why here I've decided to use only crate names, to not introduce more complexity into parsing and dict with packages bundle.